gdc_client/upload/client.py

```diff
@@ -120,6 +120,19 @@ def get_sleep_time(tries):
     return timeout * (0.5 + random.random()/2)
 
 
+def create_resume_path(file_path):
+    ''' in case the user enters a path, you want to create
+    a resume_filename.yml inside the same directory as the manifest.yml
+    '''
+
+    if '/' in file_path:
```

Comment:
There needs to be some distinction whether or not it's a path to a file or just a file, but I can definitely rephrase it another way to be more platform agnostic.